src/comfystream/scripts/setup_models.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
```

Comment:
One thing I noticed when running this script is that even if the models are already there they are going to be redownloaded. Can we improve this logic somehow to avoid extraneous bandwidth and latency when running this on upgrades vs initial deploys?

The general expected behavior is that the script is idempotent and efficient. If some model is already downloaded it should skip it (could check the hash or sth, but let's start simple and just dedup by name). So if all models are already there this should run _really_ quickly.

Reply:
Hmm for setup_nodes it's fine, we only call it on docker build anyway. I'll see what may have happened on my env that it redownloaded